### DISCUSSION TITLE
change theme

### DISCUSSION THREAD

**Discussion by ajiangstudy** [HEIGE-PCloud/Notion-Hugo#116] [Ideas]
hi，Does notion hugo support changing to other themes，such us https://github.com/CaiJimmy/hugo-theme-stack

**Comment by HEIGE-PCloud** [comment]
Yes, I think you can copy the config file, go.mod and go.sum from https://github.com/CaiJimmy/hugo-theme-stack-starter and basically you are good to go.

Notion Hugo is just a TypeScript script that automatically fetches your content from Notion API and save them to markdown files, the rest of it is just a plain Hugo project, so any theme will work.

**Reply by mannilito** [reply]
Hi there, thanks for your great work! Could you give me some guidance on which adjustments need to be done to get this theme working? https://github.com/dzello/reveal-hugo

**Reply by zakokun** [reply]
@mannilito  The author has already created an easy way to run it on local! You can create `.env` file in project root dir，include:
```
NOTION_TOKEN=ntn_xxxxxxxx
```

Then, run: `npm start`  && `hugo server`

praise him!
